lib/pages/chat/chat.dart

```diff
@@ -1294,20 +1298,37 @@ class ChatController extends State<Chat>
         context: context,
       ),
       onSendTap: () {
-        sendMedia(
-          imagePickerController,
-          room: room,
-          caption: _captionsController.text,
-        );
+        _onSendTap(imagePickerController);
         _captionsController.clear();
       },
-      onCameraPicked: (_) => sendMedia(imagePickerController, room: room),
+      onCameraPicked: (_) => _onSendTap(imagePickerController),
       captionController: _captionsController,
       focusSuggestionController: _focusSuggestionController,
       typeAheadKey: _chatMediaPickerTypeAheadKey,
     );
   }
 
+  void _onSendTap(ImagePickerGridController imagePickerController) {
+    mediaCancelTokenMapNotifier.value = {};
+    final tempMap = <String, CancelToken>{};
```

Comment:
In this case, the old cancelToken can be removed if the user sends photos twice in a row

Reply:
fixed